desktop/src/main.rs

```diff
@@ -254,9 +255,23 @@ impl App {
             .with_visible(false)
             .with_title(title)
             .with_window_icon(Some(icon))
+            .with_min_inner_size(LogicalSize::new(16, 16))
             .with_max_inner_size(LogicalSize::new(i16::MAX, i16::MAX))
             .build(&event_loop)?;
 
+        let mut max_resolution_x = 0;
+        let mut max_resolution_y = 0;
+
+        for monitor in Window::available_monitors(&window) {
+            let size = MonitorHandle::size(&monitor);
+            max_resolution_x += size.width;
+            max_resolution_y += size.height;
+        };
+
+        if max_resolution_x > 0 && max_resolution_y > 0 {
+            Window::set_max_inner_size(&window, Some(LogicalSize::new(max_resolution_x, max_resolution_y)));
```

Comment:
Can you run this before we build the window and use the value you calculate in the builder? Maybe extract it to a method that returns the size tuple